test/common/services/kibana_server/kibana_server.js

```diff
@@ -18,51 +15,7 @@ export async function KibanaServerProvider({ getService }) {
       const url = formatUrl(config.get('servers.kibana'));
       this.status = new KibanaServerStatus(url);
       this.version = new KibanaServerVersion(this.status);
-      this.uiSettings = new KibanaServerUiSettings(log, es, kibanaIndex, this.version);
-
-      lifecycle.on('beforeEachTest', async () => {
-        await this.waitForStabilization();
-      });
-    }
-
-    async waitForStabilization() {
```

Comment:
__note:__ Please, don't forget to update x-pack tests, I know many of them depend on `kbnServer.waitForStabilization`.